.github/workflows/cli-publish-snap.yml

```diff
@@ -21,3 +21,9 @@ jobs:
       with:
         snap: ${{ steps.build.outputs.snap }}
         release: edge
+    - uses: snapcore/action-publish@v1
+      env:
+        SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAP_STORE_LOGIN }}
+      with:
+        snap: ${{ steps.build.outputs.snap }}
+        release: beta
```

Comment:
Why not push to beta and just close edge, then edge should follow the same version as beta, no?